research/zipkin-opensearch-to-otel/README.md

```diff
@@ -1,22 +0,0 @@
-# Zipkin trace data from OpenSearch to Otel Trace Source
```

Comment:
What's the motivation for eliminating this research?

Reply:
It might be a little scope creep but this module is well outdated and it has dependency on otel-trace-raw-prepper.